Sources/SKTestSupport/SKSwiftPMTestWorkspace.swift

```diff
@@ -55,7 +55,7 @@ public final class SKSwiftPMTestWorkspace {
   /// Connection to the language server.
   public let testClient: TestSourceKitLSPClient
 
-  /// When `testServer` is not `nil`, the workspace will be opened in that server, otherwise a new server will be created for the workspace
+  /// When `testClient` is not `nil`, the workspace will be opened in that client's server, otherwise a new client will be created for the workspace
```

Comment:
Not really related, but I was confused by the wording here when browsing the codebase.

Reply:
Oh, I recently changed `testServer` to `testClient` and looks like I forgot that comment. Thanks for updating it.